### PR TITLE
Fix clipping issue login page

### LINKED ISSUE #7890: Login-in page is clipping in
**Describe the bug**

On the responsive view , the login page text is clipping in

**To Reproduce**

1. Go to care.ohc.in
2. select any other languages
3. switch to mobile view

**Expected behavior**

in mobile view, it should be properly responsive, irrespective of which language the user select's

**Screenshots**

image

### PR DESCRIPTION
**Pull request by sunny-thakurwar**
## Proposed Changes

- Fixes #7890
- Using word-break: break-all to break long words if possible which does not clip the log in page anymore.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by sunny-thakurwar** [comment]
@nihal467 can you please check once

File: src/Components/Auth/Login.tsx [hunk 1]
```diff
@@ -282,7 +282,7 @@ export const Login = (props: { forgot?: boolean }) => {
                   forgotPassword && "hidden",
                 )}
               >
-                <div className="mb-8 w-[300px] text-4xl font-black text-primary-600">
+                <div className="mb-8 w-[300px] break-all text-4xl font-black text-primary-600">
```
Comment (rithviknishad) [inline src/Components/Auth/Login.tsx]: @sunny-thakurwar This seems to be breaking in both mobile view and desktop view even for english. It'd be great if you could test the changes out before requesting a review 😄 

image

image

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Auth/Login.tsx (hunk 1 above)
Comment (sunny-thakurwar) [reply src/Components/Auth/Login.tsx]: @rithviknishad   Opps! i didn't checked with english language. srry for that corrected the responsiveness please check now.

**Comment by nihal467** [comment]
@sunny-thakurwar 
image

- [ ] When we select the Tamil language, it uses three rows in the normal desktop view, even though there is enough space. Please make it responsive so that it only uses two rows. Kindly comment if there are any technical limitations.

**Comment by sunny-thakurwar** [comment]
@nihal467 the text looks fine on my system. committed some changes hope this resolves the issue. if the issue persists please share the screen resolution you are testing on which it is breaking. also share browser version.  

image

**Comment by nihal467** [comment]
@sunny-thakurwar 
image

when we click the Kannada language, even though we can write it in a single row, it utilising two rows, can you check it

**Comment by sunny-thakurwar** [comment]
@nihal467 @rithviknishad  srry for the reworks. i fixed the issue mentioned in comment. can you please check now

File: src/Components/Auth/Login.tsx [hunk 2]
```diff
@@ -282,7 +282,7 @@ export const Login = (props: { forgot?: boolean }) => {
                   forgotPassword && "hidden",
                 )}
               >
-                <div className="mb-8 w-[300px] text-4xl font-black text-primary-600">
+                <div className="mb-8 w-[250px] max-w-full text-[min(10vw,1.85rem)] font-black text-primary-600 lg:w-[300px] lg:text-4xl">
```
Comment (rithviknishad) [inline src/Components/Auth/Login.tsx]: Prefer `rem` values over hardcoded pixel values. Pixel values may produce inconsistent results on different browser/screen zoom settings.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Auth/Login.tsx (hunk 2 above)
Comment (gigincg) [reply src/Components/Auth/Login.tsx]: @rithviknishad Switched to `w-64` (256px) &  `w-72` (288px)

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**